### DISCUSSION TITLE
[Feature Request]

### DISCUSSION THREAD

**Discussion by germanolleunlp** [microsoft/monaco-editor#3978] [Q&A]
### Context

- [X] This issue is not a bug report. *(please use a different template for reporting a bug)*
- [X] This issue is not a duplicate of an existing issue. *(please use the [search](https://github.com/microsoft/monaco-editor/issues) to find existing issues)*

### Description

Is there any way to be able to handle `\n` and `\r` or `\r\n` at the same time?. I added two commands to add these special characters but for some reason the editor is transforming all of them to be only one.
For example if I enter `This is\n` and then `an\r\nexample` the return will be `This is\nand\nexample`. I saw this behavior as well for any `textarea`. Is there any way to get the `raw` data for the textarea?

**Comment by Prinzhorn** [comment]
This is one of the longest standing issues with countless duplicates linking to https://github.com/microsoft/vscode/issues/127
